src/Blef.GameLogic/PokerHands/!PokerHandHierarchy.txt

```diff
@@ -1,11 +1,11 @@
-﻿HighStraightFlush - duży poker (kolor) 
-LowStraightFlush -  mały poker (kolor)
-FourOfKind - kareta (wartość)
-Flush - kolor (kolor)
-FullHouse - full (wartość, wartość)
-ThreeOfKind - trójka (wartość)
-HighStraight - duży strit
-LowStraight - mały strit
-TwoPairs - dwie pary (wartość, wartość)
-Pair - para (wartość)
-HighCard - wysoka karta (wartość)
\ No newline at end of file
+﻿HighStraightFlush - duży poker (kolor) 1 000 000 000 *S
+LowStraightFlush -  mały poker (kolor) 100 000 000 *S
+FourOfKind - kareta (wartość) 10 000 000 *R
+Flush - kolor (kolor) 1 000 000 * S
+FullHouse - full (wartość, wartość) 100 000 + 10R1 +R2
```

Comment:
R means Rank value and S means Suit value. I've assigned value to each poker hands to establish hierarchy. Each poker hand value is const value or function of components (suit or rank).
FullHouse consists of two ranks (ex first rank = three kings and second rank = two queens) so the full value of FullHouse is basic value (100 000) + 10* firstRank* rankValue + secondRank*rankValue. Simply speaking R1 is the first rank (which is stronger) and R2 is the second rank.